docs/code/getting-started.md

```diff
@@ -584,11 +600,11 @@ Let’s create a Window with a vertical Grid that contains a Button and a Label:
     grid.add (button);
     grid.add (label);
 
```

Comment:
More specific variable names?

Reply:
This isn't a change that comes from my branch